enterprise/premium-skins-and-icon-packs/bootstrap-demo.md

```diff
@@ -6,7 +6,7 @@ description: Bootstrap Demo
 keywords: skin skins icon icons bootstrap customize theme
 ---
 
-This demo shows an example form using Bootstrap together with {{site.productname}} with the _Bootstrap_ skin. It's designed to closely matches _colors, forms and buttons_ to the default Bootstrap framework
+This demo shows an example form using Bootstrap together with {{site.productname}} with the _Bootstrap_ skin and _Bootstrap_ icon pack. It's designed to closely matches _colors, forms and buttons_ to the default Bootstrap framework
```

Comment:
```suggestion
This demo shows an example form using Bootstrap together with {{site.productname}} and the {{site.companyname}} _Bootstrap_ skin and _Bootstrap_ icon pack. It's designed to closely match the _colors, forms, and buttons_ of the default Bootstrap framework UI.
```
Maybe?

Reply:
If you're changing that you'll need to change all premium skin demos, as they are all the same as my original wording.

Reply:
:facepalm: :vomiting_face: Up to you. ether accept my change so at least one is right, or ignore me and I'll fix them all one day.